packages/playground-ui/src/ds/components/DialogNew/dialog-new.css

```
@@ -0,0 +1,111 @@
+  animation-name: dialog-new-hold;
+  animation-timing-function: linear;
+  animation-fill-mode: forwards;
+}
+
+@keyframes dialog-new-hold {
+  to {
+    clip-path: inset(0);
+  }
+}
+
+@media (max-width: 22rem) {
```

Stylelint (17.14.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)